ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ConnectExecutor.java

```diff
@@ -42,7 +43,10 @@ private ConnectExecutor() { }
 
     final ConnectResponse<ConnectorInfo> response = client.create(
         createConnector.getName(),
-        Maps.transformValues(createConnector.getConfig(), l -> l.getValue().toString()));
+        ConnectTemplate.resolve(
+            Maps.transformValues(
+                createConnector.getConfig(),
+                l -> l != null ? l.getValue().toString() : null)));
```

Comment:
Why would this ever be null?

Reply:
never... it's just that intelliJ complains when I don't check for it because transformValues has an annotation (`@NullableDecl`)